### PR TITLE
Added the windows build to travis CI

### PR DESCRIPTION
**Pull request by pratik0316**
Addressing to issue #136

### CODE REVIEW

**Review by steven-esser — CHANGES_REQUESTED** [review]
Please rebase this to a single commit with detailed commit message.

File: .travis.yml [hunk 1]
```diff
@@ -41,20 +29,24 @@ matrix:
...
-notifications:
-  irc:
-    channels:
-      - "chat.freenode.net#aboutcode"
-  on_sucess: change
-  on_failure: always
-  use_notice: true
-  skip_join: true
-  template:
-    - "%{repository_slug}#%{build_number} (%{branch}-%{commit}:%{author})-%{message}- %{build_url}"
```
Comment (steven-esser) [inline .travis.yml]: Why remove this?

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (pratik0316) [reply .travis.yml]: actually I was trying to sort out the error in Travis CI ,I am currently working over it.

**Review by steven-esser — CHANGES_REQUESTED** [review]
Please rebase this to a single commit.

**Comment by pratik0316** [comment]
@majurg I am closing this branch for now, I have opened a new [PR](https://github.com/nexB/deltacode/pull/150) for it(3  of windows build are failing on it )

**Pull request closed without merging**